Dapper/SqlMapper.Settings.cs

```diff
@@ -121,6 +121,11 @@ public static long FetchSize
                 }
             }
 
+            /// <summary>
+            /// If true, Dapper will try to determine if the current sql statement is ole db statement and not filter
+            /// </summary>
+            public static bool OleDbCheckEnabled { get; set; } = true;

```

Comment:
not loving this name; naming is hard; maybe `SupportLegacyParameterTokens` ?

also, more comments would help here, I think; perhaps:

``` txt
Indicates whether single-character parameter tokens (<c>?</c> etc) will be detected and used where possible;
this feature is not recommended and will be disabled by default in future versions; where possible, prefer named
parameters (<c>@yourParam</c> etc) or Dapper's "pseudo-positional" parameters (<c>?yourParam? etc</c>).
```

Reply:
Yeah, naming is hard